### PR TITLE
OCPBUGS-34082: baremetal: check if HardwareDetails is nil

### PR DESCRIPTION
**Pull request by honza**

### CODE REVIEW

File: pkg/infrastructure/baremetal/variables.go [hunk 1]
```diff
@@ -99,6 +99,13 @@ func getMasterAddresses(dir string) ([]string, error) {
 	}
 
 	for _, bmh := range hosts {
+		logrus.Debug("  bmh:", bmh.Name)
+
+		if bmh.Status.HardwareDetails == nil {
+			logrus.Debug("    HardwareDetails nil, skipping")
```
[the remark this answers is not in the document]
Comment (dtantsur) [reply pkg/infrastructure/baremetal/variables.go:105]: maybe you meant the BMH name?

**Review by r4f4 — COMMENTED** [review]
/approve

**Comment by r4f4** [comment]
Seems to be running fine now. If it fails, I'll override.

**Comment by r4f4** [comment]
/override ci/prow/okd-images
Not related to PR changes.

**Pull request merged**